src/css/styles.scss

```diff
@@ -569,6 +569,34 @@
       display: none;
     }
   }
+  .menu-container {
+    .menu-item {
+      &.active-collab {
+        background-color: #ecfdf5;
+        color: #064e3c;
+      }
+      &.ExcalidrawPlus {
+        color: var(--color-promo);
+      }
+      &.language {
+        padding: 0.0625rem;
+      }
+    }
+    .menu-group.menu-footer {
+      display: flex;
+      flex-direction: column;
+      row-gap: 0.5rem;
+    }
+  }
+
+  &.theme--dark {
+    .menu-item {
+      &.active-collab {
+        background-color: #064e3c;
+        color: #ecfdf5;
+      }
+    }
+  }
 }
```

Comment:
move to `Menu.scss`

Reply:
This is specific to the Excalidraw app hence cannot be part of `Menu.scss`

Reply:
I mistakenly left it inside excalidraw package code, I have moved it to excalidraw app instead now